### ISSUE TITLE
css or sass ?

### ISSUE THREAD

**Issue by freedomtrain** [kakauandme/beats1.media#1]
Trying to have this great service up and running and have run into an issue.
There seems to be no css directory included in the package ?
I see a sass directory but the code seems to reference /css

Can you let me know if I have missed something ?

Thanks

**Comment by kakauandme** [comment]
Hey, 

You have to compile SASS files into CSS. 

Also you will need to create a DB to make website work.

I'll add installation notes to the repository soon.

Cheers

**Comment by freedomtrain** [comment]
Thanks !!!!

On March 4, 2016 at 1:45:38 PM, Kirill Kliavin (notifications@github.com) wrote:

Hey,

You have to compile SASS files into CSS.

Also you will need to create a DB to make website work.

I'll add installation notes to the repository soon.

Cheers

**Comment by freedomtrain** [comment]
I compiled the SASS into CSS
Now I receive this error message 

[[Fri Mar 04 23:11:20 2016] [error] PHP Notice:  Trying to get property of non-object in /srv/www/fox.leola.me/now.php on line 34
[Fri Mar 04 23:11:20 2016] [error] PHP Warning:  Creating default object from empty value in /srv/www/fox.leola.me/now.php on line 34
[Fri Mar 04 23:11:21 2016] [error] PHP Notice:  Trying to get property of non-object in /srv/www/fox.leola.me/now.php on line 34
[Fri Mar 04 23:11:21 2016] [error] PHP Warning:  Creating default object from empty value in /srv/www/fox.leola.me/now.php on line 34

Any help will be greeting appreciated !!!!

Thanks William 

 

On March 4, 2016 at 2:07:27 PM, william Dyson (william@ida.coop) wrote:

Thanks !!!!

On March 4, 2016 at 1:45:38 PM, Kirill Kliavin (notifications@github.com) wrote:

Hey,

You have to compile SASS files into CSS.

Also you will need to create a DB to make website work.

I'll add installation notes to the repository soon.

Cheers

**Comment by kakauandme** [comment]
Yep database is missing.

I've just pushed DB dump file.

Just create a mysql database and import data from **beats1_media.sql**, also make sure 

```
$servername = "";
$username = "";
$password = "";
$dbname = "";
```

variables  in **variables.php** are adjusted for your local setup.

**Comment by freedomtrain** [comment]
That did it !!!!!

Thanks so much !!!!!

 

On March 4, 2016 at 4:02:25 PM, Kirill Kliavin (notifications@github.com) wrote:

Yep data base is missing.

I've just pushed DB dump file.

Just create a mysql database and import data from beats1_media.sql, also make sure

$servername = "";
$username = "";
$password = "";
$dbname = "";

variables in variables.php are adjusted for your local setup.

**Comment by freedomtrain** [comment]
Kirill
All is working expect playing now….For some reason I’am receiving the following error 

 Trying to get property of non-object in /srv/www/fox.leola.me/now.php on line 34

Thanks for your help 

William 

On March 6, 2016 at 3:16:53 AM, Kirill Kliavin (notifications@github.com) wrote:
Closed #1.

**Comment by kakauandme** [comment]
Make sure you run **crawler.php** at least once a day. It pulls data from Apple's stream and adds it to the DB.  **now.php** displays last record from the DB.
